docs/curriculum/detailed-schedule.md

```diff
@@ -0,0 +1,80 @@
+| 9:00 | 9:15 | Welcome to the OSG User School                                                | Tim C. |
+| 9:15 | 10:15 | Lecture & demo: Introduction to HTC and HTCondor                              | Lauren |
+| 10:15 | 10:30 | Break                                                                         | — |
+| 10:30 | 12:00 | Hands-on: Logging in, submitting single jobs, HTCondor tools                  | staff |
+| 12:00 | 13:00 | Lunch                                                                         | — |
+| 13:00 | 13:45 | Lecture & demo: More HTCondor                                                 | Rachel |
+| 13:45 | 14:45 | Hands-on: (catch up), submitting multiple jobs, etc.                          | staff |
+| 14:45 | 15:00 | Break                                                                         | — |
+| 15:00 | 15:15 | Lecture: Setting goals for the School and beyond                              | Christina |
+| 15:15 | 17:00 | Hands-on: Goals and unfinished exercises <br> Individual consultations (some) | staff |
+
+# July 26 (Tuesday)
+
+| Start | End | Event                                                                            | Instructor |
+|-------|-------|----------------------------------------------------------------------------------|------------------------|
+| 8:00 | 8:45 | Breakfast                                                                        | — |
+| 9:00 | 9:45 | Lecture & demo: Introduction to OSG and the OSPool                               | Tim C. |
+| 9:45 | 10:30 | Hands-on: Using the OSPool                                                       | staff |
+| 10:30 | 10:45 | Break                                                                            | — |
+| 10:45 | 11:30 | Lecture & demo: Troubleshooting jobs                                             |  |
+| 11:30 | 12:15 | Hands-on: Basic troubleshooting and repair                                       | staff |
+| 12:15 | 13:15 | Lunch                                                                            | — |
+| 13:15 | 14:30 | Interactive: High Throughput Computing in action                                 | staff |
+| 14:30 | 14:45 | Break                                                                            | — |
+| 14:45 | 15:30 | Lecture & demo: Software portability                                             | Christina |
+| 15:30 | 17:00 | Hands-on: Software and unfinished exercises <br> Individual consultations (some) | staff |
+
+# July 27 (Wednesday)
+
+| Start | End                            | Event                                                           | Instructor |
+|-------|--------------------------------|-----------------------------------------------------------------|------------|
+| 8:00  | 9:00 | Breakfast at Gordon Avenue Market                               | —          |
+| 8:45  | 9:15 | Walk to Computer Sciences                                       | —          |
```

Comment:
9:15 End to 9:30 Start

Reply:
Good catch! However, this one is on purpose.